### PR TITLE
Add groups to session too when creating session from bearer token

### PR DESCRIPTION
**Pull request by adriananeci**
## Description

If using `--skip-jwt-bearer-tokens` together with `--allowed-goup` and passing a bearer token that contains the `groups` claim, the authentication is failing with `[AuthFailure] Invalid authorization via session` because the groups are not populated into the session.

This PR extends token to session flow to also populate the groups for the session with the token groups claim.

## Motivation and Context

Ability to run `--skip-jwt-bearer-tokens` together with `--allowed-goup`

## Checklist:

- [x] My change requires a change to the documentation or CHANGELOG.
- [x] I have updated the documentation/CHANGELOG accordingly.
- [x] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

**Review by braunsonm — DISMISSED** [review]
This looks good thanks! Would like to see what our other maintainers think before merging.

**Comment by yburke94** [comment]
OOC, once this is merged, what would the timeline be before it ends up in a release?

**Comment by JoelSpeed** [comment]
Changes look good, please add a changelog entry and I'm happy to merge it

Timeline for a release is not currently set, we have a few bugs we need to work through and will cut a release once we have those resolved

**Comment by adriananeci** [comment]
> please add a changelog entry and I'm happy to merge it

Done.

**Review by JoelSpeed — APPROVED** [review]

**Comment by yburke94** [comment]
@JoelSpeed Are we any closer to getting a release cut for this? Apologies for asking again but I have no easy way of building this project from source in my company and require this feature.

**Comment by JoelSpeed** [comment]
Please refer to #1624, I've just added a comment to that issue which explains what's happening

**Pull request merged**
